common/datasets/switchboard/train.py

```diff
@@ -0,0 +1,188 @@
+    """
+    swb_trans_and_dict = DownloadSwitchboardTranscriptionAndDictJob()
+    swb_trans_and_dict.add_alias(
+        os.path.join(subdir_prefix, "download_trans_and_dict_job")
+    )
+
+    speakers_list = get_speakers_list_legacy(subdir_prefix=subdir_prefix)
+    corpus = CreateSwitchboardBlissCorpusJob(
+        audio_dir=audio_dir,
+        trans_dir=swb_trans_and_dict.out_trans_dir,
+        speakers_list_file=speakers_list,
+        skip_empty_ldc_file=skip_empty_ldc_file,
+    )
+    corpus.add_alias(
+        os.path.join(
+            subdir_prefix,
+            "create_train_corpus_job_skip_empty_%s" % str(skip_empty_ldc_file),
+        )
+    )
+
+    return corpus.out_corpus
+
+
+def get_train_bliss_corpus_ldc(subdir_prefix: str = SUBDIR_PREFIX) -> tk.Path:
+    """
+    Switchboard-1 training corpus based on the original LDC file
+    Uses i6-custom text processing, and the transcriptions are fully
+    identical to the "i6-legacy" version.
+
+    :param subdir_prefix:
+    :return:
+    """
+    swb_trans_and_dict = DownloadSwitchboardTranscriptionAndDictJob()
+    swb_trans_and_dict.add_alias(
+        os.path.join(subdir_prefix, "download_trans_and_dict_job")
+    )
+
+    audio_dir = SwitchboardSphereToWaveJob(
+        sph_audio_folder=SWITCHBOARD1_PATH,
+    ).out_wave_audio_folder
+
+    speaker_list_file = get_speakers_list_ldc(subdir_prefix)
+
+    corpus = CreateSwitchboardBlissCorpusJob(
+        audio_dir=audio_dir,
+        trans_dir=swb_trans_and_dict.out_trans_dir,
+        speakers_list_file=speaker_list_file,
+        skip_empty_ldc_file=True,
+        lowercase=True,
+    )
+    corpus.add_alias(os.path.join(subdir_prefix, "create_train_corpus_job"))
+
+    return corpus.out_corpus
+
+
+def get_train_bliss_corpus_i6_legacy(subdir_prefix: str = SUBDIR_PREFIX) -> tk.Path:
+    """
+    i6-internal Switchboard-1 training corpus
+
+    :param subdir_prefix:
+    :return:
+    """
+    subdir_prefix = os.path.join(subdir_prefix, "Switchboard-i6-legacy")
+    train_bliss_corpus = get_train_bliss_corpus(
+        SWITCHBOARD1_LEGACY_PATH, subdir_prefix=subdir_prefix
+    )
+    return train_bliss_corpus
+
+
+def get_train_corpus_object_ldc(subdir_prefix: str = SUBDIR_PREFIX):
+    """
+    :param subdir_prefix:
+    :return:
+    """
+    corpus_object = CorpusObject()
+    corpus_object.corpus_file = get_train_bliss_corpus_ldc(subdir_prefix=subdir_prefix)
+    corpus_object.audio_format = "wav"
+    corpus_object.audio_dir = None
+    corpus_object.duration = 311.61
+
+    return corpus_object
+
+
+def get_train_corpus_object_i6_legacy(
+    subdir_prefix: str = SUBDIR_PREFIX,
+) -> CorpusObject:
+    """
+    :param subdir_prefix:
+    :return:
+    """
+    corpus_object = CorpusObject()
+    corpus_object.corpus_file = get_train_bliss_corpus_i6_legacy(
+        subdir_prefix=subdir_prefix
+    )
+    corpus_object.audio_format = "wav"
+    corpus_object.audio_dir = None
+    corpus_object.duration = 311.78
```

Comment:
Is there a reason this is not in the constants?

Reply:
here it is a deprecated custom corpus so we keep it out of the constants.